app/views/layouts/application.html.erb

```diff
@@ -90,24 +90,24 @@
   <div class="container">
     <hr>
 
-    <div class="row">
+    <div id="footer-authoring" class="row">
```

Comment:
It is ok to provide a hook for customizing the authoring. However, please notice that as [Share Alike](https://creativecommons.org/licenses/by-sa/2.5/) License states, you are obliged to display who is the author.